sway-core/src/asm_lang/allocated_ops.rs

```diff
@@ -67,46 +68,42 @@ pub(crate) enum AllocatedOpcode {
     EXP(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     EXPI(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
     GT(AllocatedRegister, AllocatedRegister, AllocatedRegister),
-    GTF(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
     LT(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     MLOG(AllocatedRegister, AllocatedRegister, AllocatedRegister),
-    MROO(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     MOD(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     MODI(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
     MOVE(AllocatedRegister, AllocatedRegister),
     MOVI(AllocatedRegister, VirtualImmediate18),
+    MROO(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     MUL(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     MULI(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
+    NOOP,
     NOT(AllocatedRegister, AllocatedRegister),
     OR(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     ORI(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
     SLL(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     SLLI(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
-    SMO(
-        AllocatedRegister,
-        AllocatedRegister,
-        AllocatedRegister,
-        AllocatedRegister,
-    ),
     SRL(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     SRLI(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
     SUB(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     SUBI(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
     XOR(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     XORI(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
+
+    /* Conrol Flow Instructions */
+    JMP(AllocatedRegister),
     JI(VirtualImmediate24),
+    JNE(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     JNEI(AllocatedRegister, AllocatedRegister, VirtualImmediate12),
     JNZI(AllocatedRegister, VirtualImmediate18),
-    JMP(AllocatedRegister),
-    JNE(AllocatedRegister, AllocatedRegister, AllocatedRegister),
     RET(AllocatedRegister),
```

Comment:
If you're going to move `RETD` then you might as well move `RET`, they're pretty much the same.  I had `JMP` and `JNE` together because they're dynamic jumps while the others are all immediate.  If you're going for alpha then `JMP` comes after `JI`.

Reply:
I honestly just following the order and the grouping in the spec, which is not 100% alphabetical.